otcextensions/osclient/auto_scaling/v1/group.py

```diff
@@ -337,84 +369,83 @@ def get_parser(self, prog_name):
             help=_('AS Group name or ID')
         )
         parser.add_argument(
-            '--desire_instance_number',
+            '--desire-instance-number',
             metavar='<desire_instance_number>',
             type=int,
             help=_('Desired number of instances')
         )
         parser.add_argument(
-            '--min_instance_number',
+            '--min-instance-number',
             metavar='<min_instance_number>',
             type=int,
             help=_('Minimal number of instances')
         )
         parser.add_argument(
-            '--max_instance_number',
+            '--max-instance-number',
             metavar='<max_instance_number>',
             type=int,
             help=_('Maximal number of instances')
         )
         parser.add_argument(
-            '--cool_down_time',
+            '--cool-down-time',
             metavar='<cool_down_time>',
             type=int,
             help=_('Specifies cooling duration in seconds')
         )
         parser.add_argument(
-            '--lb_listener_id',
+            '--lb-listener-id',
             metavar='<lb_listener_id>',
             help=_('Specifies ELB Listener ID')
         )
         parser.add_argument(
-            '--lbaas_listener',
+            '--lbaas-listener',
             metavar='<lbaas_listener>',
             action='append',
             help=_('Specifies ULB Listener Information in format: '
                    'ID:PORT:Weight(optional) '
                    '(Repeat multiple times, up to 3 times)')
         )
         parser.add_argument(
-            '--availability_zone',
+            '--availability-zone',
             metavar='<availability_zone>',
             action='append',
             help=_('Specifies the availability zones information '
                    '(Repeat multiple times)')
         )
         parser.add_argument(
             '--subnetwork',
```

Comment:
can you please check this param name? we normally use "subnet" if talk really about subnet. Additionally help text here is still confusing

Reply:
This is the network ID, not the subnet. Shall I rename it?

Reply:
yes, but we have already here also network-id, which is mapped to vpc_id - to be fixed also. Same renaming should be done also in UpdateAutoScalingGroup